### PR TITLE
C# Webapp setup with call to aylien for sentiment analysis

### PR DESCRIPTION
**Pull request by Keshav-Pandey**
Setup a web app for the project.
Added functionality to call Aylien sentiment analysis

### CODE REVIEW

File: CognitoBot/Controllers/WeatherForecastController.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
+﻿using System;
```
Comment (monikajengaria) [inline CognitoBot/Controllers/WeatherForecastController.cs]: Why do we have weather forecast controller?

Comment (monikajengaria) [inline CognitoBot/Controllers/WeatherForecastController.cs]: Why do we have a weatherforecast controller

File: CognitoBot/WeatherForecast.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+using System;
```
Comment (monikajengaria) [inline CognitoBot/WeatherForecast.cs]: Why do we have weather forecast class

**Review by monikajengaria — CHANGES_REQUESTED** [review]

File: CognitoBot/Controllers/WeatherForecastController.cs (hunk 1 above)
Comment (Keshav-Pandey) [reply CognitoBot/Controllers/WeatherForecastController.cs]: I was using it as a test bed for the sentiment analysis class

**Review by monikajengaria — COMMENTED** [review]
Looks good

**Review by monikajengaria — APPROVED** [review]

**Pull request merged**